elements/pl-multiple-choice/pl-multiple-choice.py

```diff
@@ -142,12 +158,22 @@ def prepare(element_html, data):
     # - nota and aota disabled
     number_answers = min(min(1, len_correct) + len_incorrect, number_answers)
 
-    if aota_correct:
+    if nota == 'random' or aota == 'random':
+        # Either 'None of the above' or 'All of the above' is correct
+        # with probability 1/(number_correct + nota + aota).
+        prob_space = len_correct + (1 if nota == 'random' else 0) + (1 if aota == 'random' else 0)
```

Comment:
this `1 if ... else 0` is clearer to me than the implicit conversion from boolean to int up above; would you mind making the change there too?

Reply:
Done in eb2d5b3. I did some rearranging as well to make it easier to follow.